src/services/branch-manager.js

```diff
@@ -1,3 +1,13 @@
+const P = require('bluebird');
+const agent = require('superagent-promise')(require('superagent'), P);
```

Comment:
[Suggestion] Not mandatory (Since I also add it in my PR), but it would be nice to remove `bluebird` here I guess, since `Promise` should work ? (Didn't test)